src/dev-mock-service/README.md

```diff
@@ -0,0 +1,47 @@
+## Dev Mock Service
```

Comment:
:no_bell: suggest consider renaming to just "mocks". "dev" and "service" aren't buying me much. same thing with the subdirectory names which could be like mock-reddit instead of reddit-api-mock-service.